code/modules/mob/living/living_defense.dm

```diff
@@ -705,7 +705,7 @@
 			log_combat(src, target, "shoved", "knocking them down[weapon ? " with [weapon]" : ""]")
 			return
 
-	if(shove_flags & SHOVE_CAN_KICK_SIDE) //KICK HIM IN THE NUTS
+	if((shove_flags & SHOVE_CAN_KICK_SIDE) && !HAS_TRAIT_FROM(target, TRAIT_IWASBATONED, REF(src))) //KICK HIM IN THE NUTS
```

Comment:
This check should be in `/mob/living/carbon/get_shove_flags`, IE, it should not return `SHOVE_CAN_KICK_SIDE` if they have the trait